app/services/editions/publishing/IssuePublishedSNSMessageParser.scala

```diff
@@ -0,0 +1,26 @@
+package services.editions.publishing
+
+import com.amazonaws.services.sqs.model.Message
+import logging.Logging
+import play.api.libs.json.Json
+
+import scala.util.Try
+
+object IssuePublishedSNSMessageParser extends Logging {
+
+  import IssuePublishedSQSMsgFormatter._
+
+  def parseMsg(msgWithSNSNotification: Message): Try[IssuePublishedEventWrapper] = {
+    logger.info("read new publish events")
+    import Json.{fromJson, parse}
```

Comment:
any reason to have a local import here, given we have `import play.api.libs.json.Json` up top?

Reply:
sure

Reply:
done